inst/include/Rcpp/sugar/matrix/lower_tri.h

```diff
@@ -1,8 +1,9 @@
-// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; tab-width: 8 -*-
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
```

Comment:
Thanks for catching that here.  Slowly changing them myself whenever I see one in a file I use.